bundles/org.openhab.binding.vesync/README.md

```diff
@@ -295,6 +310,23 @@ Frame {
 }
 ```
 
+#### Air Humidifier Oasis Mist Smart Model
+
+```
```

Comment:
```suggestion
```perl
```

Reply:
Thank you @jlaur, I hadn't seen that in any of the other bindings I've looked at. After searching I see its common :) its now updated to the convention for the sitemap examples to include the perl tag.